azure-pipelines.yml

```diff
@@ -69,6 +75,17 @@ jobs:
     pool:
       vmImage: 'ubuntu-latest'
     steps:
+      - task: UsePythonVersion@0
+        displayName: 'Set Python version'
+        inputs:
+          versionSpec: '2.7.18'
+          addToPath: true
+      - task: JavaToolInstaller@0
+        displayName: 'Set Java version'
+        inputs:
+          versionSpec: '8'
+          jdkArchitectureOption: 'x64'
+          jdkSourceOption: 'PreInstalled'
       - script: sbt ++2.11.12 clean
```

Comment:
@imback82 in run-tests.py, it names hyperspace package as:
```
package = "com.microsoft.hyperspace:hyperspace-core_2.12:" + version
```
But here it's 2.11. Is there any reason for this?

Reply:
Probably a byproduct of https://github.com/delta-io/delta/blob/4277443703c5ab59a567c1e80189bbcdb7495817/python/run-tests.py#L67. We can fix this if this doesn't work.

Reply:
it's working anyway but I was just curious.